amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -276,6 +318,82 @@ function compareAll(){
 										   value="&nbsp;&nbsp;<digi:trn>Compare All</digi:trn>&nbsp;&nbsp;"
 										   style="cursor: pointer; font-style: italic; float: right; margin: 0.5% 1.5% 0.5%;">
 								</c:if> <br>
+								
+				  
+				  <span style="cursor:pointer;font-style: italic;float:right;" onClick="toggleFilterSettings();" id="displayFilterButton">
+				  <digi:trn key="aim:Showfilteroptions">Show Filter options</digi:trn> &gt;&gt;</span>
+                                &nbsp;<br>
+								<div style="display:none;background-color:#ffffff;padding:2px" id="currentFilterSettings" >
+                                 <table cellpadding="2" cellspacing="2" border="0" width="250px">
+                                 <tr>
+                                 	<td align="right">
+                                 	<strong><digi:trn>Filter by User:</digi:trn>&nbsp;&nbsp;</strong>
+                                	</td>
+                                	<td>
+                                    <html:select property="selectedUser" styleClass="inp-text" styleId="userId">
+                                 	<html:option value="-1">Select User </html:option>
+                                 	<html:optionsCollection property="userList" value="id" label="name"></html:optionsCollection>
+                                 	</html:select>
+                                 	</td> 
+                                 	<tr>
+                                 	<td align="right">
+                                 	<strong><digi:trn>Filter by Team:</digi:trn>&nbsp;&nbsp;</strong>
+                                	</td>
+                                	<td>
+                                 	<html:select property="filteredTeam" styleClass="inp-text" styleId="teamId">
+                                 	<html:option value="-1">Select Team </html:option>
+                                	<html:options property="teamList"></html:options>
+                                 	</html:select>
+                                 	</td>
+                                 	</tr> 
+                                    <tr>
+                                 	<td align="right">
+                                 	<c:set var="dateTr">
+							        <digi:trn>Filter by date:</digi:trn>
+						            </c:set>
+				  <c:if test="${aimAuditLoggerManagerForm.withDate==true }">
+						<input type="checkbox" id="date" onchange="dateFilter()" checked="checked">${dateTr}

```

Comment:
I would avoid this to make the screen less complex

Reply:
I can't remove this b/c I used this in my code to check if withDate is selected.
b/c It's not updating the value of filterByDate in the action Form

Reply:
ok i will check why its not updating